src/ripple/protocol/STParsedJSON.h

```diff
@@ -45,7 +45,7 @@ class STParsedJSONObject
     ~STParsedJSONObject () = default;
 
     /** The STObject if the parse was successful. */
-    std::unique_ptr <STObject> object;
+    boost::optional <STObject> object;
```

Comment:
Suggestion: Change all of these member object names to use the `name_` convention. eg. `object_`, `array_`, `error_`.